config_defaults/subtests/docker_cli/run.ini

```diff
@@ -54,6 +54,20 @@ last_name_sticks = yes
 names_count = 1000
 exit_status = 0
 
+[docker_cli/run/run_tmpfs]
+cmd = 'sleep 5s'
+# command to execute in running container
+# passed as argument to exec
+fndmnt = findmnt -n %(tmpfs_path)s 
+# Test will find container by CID instead of name, CID must appear
+# on stdout.
+run_options_csv = --detach=true,--tmpfs %(tmpfs_path)s
```

Comment:
Nit: trailing whitespace. I don't know why github isn't griping about it; @cevich any ideas if there's a setting for that in the repo config? Anyway, @afomm, if you use emacs search for Noah Friedman's nuke-trailing-whitespace.el; if vi, any search for vim+whitespace will offer advice on detecting/removing it.